diff --git a/node_modules/resend/build/src/resend.js b/node_modules/resend/build/src/resend.js
index c557fc9..80ab16a 100644
--- a/node_modules/resend/build/src/resend.js
+++ b/node_modules/resend/build/src/resend.js
@@ -30,7 +30,8 @@ class Resend {
         if (!key) {
             this.key = process.env.RESEND_API_KEY;
             if (!this.key) {
-                throw new Error('Missing API key. Pass it to the constructor `new Resend("re_123")`');
+                // Do not throw an error on build time!
+                // throw new Error('Missing API key. Pass it to the constructor `new Resend("re_123")`');
             }
         }
         this.headers = {
